_data/fa-owners.yml

```diff
@@ -0,0 +1,353 @@
+'API definition':
+  - area: 'API definition'
+    label: api-definition
+    owner:
+    - sq-lb-epitome
+    repos:
+    - loopback-api-definition
+    - loopback-component-explorer
+    - loopback-swagger
+
+'Boot':
+  - area: Boot
+    label: boot
+    owner:
+    - sq-lb-asteroid
+    repos:
+    - loopback-boot
+    - loopback-example-app-logic
+    smes:
+    - davidcheung
```

Comment:
The intent is sugar to show it's a handle type link without the mouseover -- ie. superkhau vs @superkhau, which tells us that is our handle (for GitHub, Twitter, etc), even if they decide not to click on it.